junction/src/test/java/org/killbill/billing/junction/plumbing/billing/json/TestBillingEventSetJson.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2014-2019 Groupon, Inc
+ * Copyright 2014-2019 The Billing Project, LLC
+ *
+ * The Billing Project licenses this file to you under the Apache License, version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at:
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.killbill.billing.junction.plumbing.billing.json;
+
+import java.math.BigDecimal;
+import java.util.SortedSet;
+import java.util.TreeSet;
+import java.util.UUID;
+
+import org.joda.time.DateTime;
+import org.killbill.billing.junction.BillingEvent;
+import org.killbill.billing.junction.BillingEventSet;
+import org.killbill.billing.junction.JunctionTestSuiteNoDB;
+import org.killbill.billing.junction.plumbing.billing.DefaultBillingEventSet;
+import org.killbill.billing.junction.plumbing.billing.json.BillingEventSetJson.SubscriptionBillingEventJson;
+import org.killbill.billing.subscription.api.SubscriptionBaseTransitionType;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.joda.JodaModule;
+import com.google.inject.Inject;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertTrue;
+
+public class TestBillingEventSetJson extends JunctionTestSuiteNoDB {
+
+    @Inject
+    protected ObjectMapper mapper;
+
+    @BeforeClass(groups = "fast")
+    public void setup() {
+        mapper.registerModule(new JodaModule());
```

Comment:
This would potentially affect other tests since the mapper is injected. Could you just create a new one for this test instead?